docs/developer/git.md

```diff
@@ -0,0 +1,104 @@
+# Git help
+
+## Creating new branches
```

Comment:
This only works for people in the collaborators section; I don't think we should offer guidance here as it is not accurate for all users.